readme.txt

```diff
@@ -35,6 +35,9 @@ Crouton was forked from BMLT Tabbed UI plugin in 2018.  This plugin provides a T
 https://demo.bmlt.app/crouton
 
 == Changelog ==
+= 3.15.0
+* Added dropdown for Days of the Week, optionally can be turned off with has_days="0".
```

Comment:
If this is optional and off by default  can this wording be flipped for turned on has days = 1

Reply:
It's on by default.

Reply:
My bad totes meant to respond. I think so. On by default seems silly with the day tabs.

Reply:
Also it's a "breaking" change for lack of better term. Like I'd rather have people explicitly turn it on than explicitly turn it off